client/src/components/MultiJurisdictionAudit/RoundManagement/useAuditSettingsJurisdictionAdmin.ts

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react'
+import { toast } from 'react-toastify'
+import { api } from '../../utilities'
+import { IAuditSettings } from '../../../types'
+
+const defaultValues: IAuditSettings = {
+  state: null,
+  electionName: null,
+  online: null,
+  randomSeed: null,
+  riskLimit: null,
+}
+
+const getSettings = async (
+  electionId: string,
+  jurisdictionId: string
+): Promise<IAuditSettings | null> => {
+  try {
+    const response: IAuditSettings = await api(
+      `/election/${electionId}/jurisdiction/${jurisdictionId}/settings`
+    )
+    return response
+  } catch (err) {
+    toast.error(err.message)
+    return null
+  }
+}
+
+const useAuditSettingsJurisdictionAdmin = (
+  electionId: string,
+  jurisdictionId: string
+): IAuditSettings => {
+  const [settings, setSettings] = useState<IAuditSettings>(defaultValues)
+
+  useEffect(() => {
+    ;(async () => {
+      const newSettings = await getSettings(electionId, jurisdictionId)
+      setSettings(newSettings || defaultValues)
```

Comment:
why return default values here as opposed to just whatever the response is like you do in the `useContestsJurisdictionAdmin` hook below?

Reply:
@jonahkagan Because I'm destructuring the `online` value from it to use, so it makes things simpler in the other component to always have that value on the object, even if it's defined as `null` to indicate it's still loading.

Reply:
that makes sense. might be simpler overall to follow the pattern of the other hooks and destructure the `online` value after you check to see if the whole response is `null`. that way a `null` value for a setting doesn't get confused with it still loading. it's a hypothetical issue for now so not a big deal